registry/quilt_server/const.py

```diff
@@ -22,17 +22,15 @@ class PaymentPlan(Enum):
 
 FTS_LANGUAGE = 'english'
 
-BAD_NAMES = set([
+BAD_NAMES = frozenset([
     TEAM,
     PUBLIC,
     'anonymous',
     'quilt'
 ])
 
-def blacklisted_name(username):
-    return username in BAD_NAMES
-
 ACTIVATE_SALT = 'activate'
-PASSWORD_RESET_SALT = 'reset'
+CODE_TTL_DEFAULT = {'minutes': 10}
```

Comment:
Not really sure if this is better than just using seconds (and we already have `MAX_LINK_AGE` in seconds below)... But I don't have a strong opinion here, so whatever.